staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -68,27 +91,27 @@ func (o *AuditLogOptions) ApplyTo(c *server.Config) error {
 			}
 			c.AuditPolicyChecker = policy.NewChecker(p)
 		}
+		if o.WebhookFile != "" {
+			// TODO(ericchiang): How do we enable non-batch mode? Maybe a kubeconfig extension? Do we
+			// need to register a new type for the kubeconfig loader to parse?
+			batchMode := true
```

Comment:
Nvm, the parsing works fine without registering a type. Was hoping to make this configurable through the kubeconfig:

```yaml
apiVersion: v1
kind: Config

extensions:
- name: audit.k8s.io/webhook-config
  extension:
    mode: batch # operate in batch mode.

# clusters refers to the remote service.
clusters:
  - name: name-of-remote-authz-service
    cluster:
# ...

# users refers to the API Server's webhook configuration.
users:
  - name: name-of-api-server
    user:
# ...
```

Does this sound reasonable or too much of a hack? Should we just add another flag?